content/docs/10.configuration-guide/url.md

```diff
@@ -86,3 +86,38 @@ location / {
 }
 ```
 
+Should you wish to access Kestra via a separate context path via the reverse proxy, a change will be required in the Micronaut settings of Kestra.
+
+For instance, say I wish to access the Kestra UI through mycompany.com/kestra, add the following configuration to your Kestra startup configuration:
+
+```yaml
+micronaut:
+  server:
+    context-path: "/kestra"
+```
+
+Then, modify your above nginx configuration to the following
+
+```nginx
```

Comment:
`nginx` isn't a supported Markdown language so comes up on the website all in one colour. Changing to bash seems to get the right sort of syntax highlighting

```suggestion
```bash
```
![Screenshot 2024-05-16 at 14 38 38](https://github.com/kestra-io/docs/assets/34094921/5a590a9b-9adf-4940-b089-94bf419a37b2)